logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/SendAssetMessageUseCase.kt

```diff
@@ -150,7 +150,8 @@ private fun provideAssetMessageContent(
         assetDomain = null,  // TODO(assets): fill in the assetDomain, it's returned by the BE when uploading an asset.
         assetToken = assetId.assetToken
     ),
-    // Asset is already in our local storage and therefore accessible
+    // Asset is already in our local storage and therefore accessible but until we don't download it to external storage the asset will only
+    // be treated as "DOWNLOADED_INTERNALLY"
```

Comment:
`DOWNLOADED`, `SAVED_INTERNALLY`, or just "as downloaded internally", so it doesn't look like an enum value that doesn't exist.